chromium_src/chrome/browser/ui/views/location_bar/location_bar_view.cc

```diff
@@ -36,6 +36,15 @@
     add_trailing_decoration(item, /*intra_item_padding=*/0);    \
   }
 
+#define BRAVE_LAYOUT_HANDLE_SEARCH_PROMOTION_BUTTON_VISIBILITY              \
+  }                                                                         \
+  else if (GetSearchPromotionButton() &&                                    \
```

Comment:
Can't avoid this presubmit error. @goodov can you advice how should I handle this?
```
brave/chromium_src/chrome/browser/ui/views/location_bar/location_bar_view.cc:41:  (cpplint) If an else has a brace on one side, it should have it on both  [readability/braces] [5]
```

Reply:
`// NOLINT` ?